### PR TITLE
13 remove unused names

### PR DESCRIPTION
**Pull request by HedgehogHolly**
I tested it in the terminal and it seems to work. 

Hopefully, I've removed the right things?

### CODE REVIEW

File: names.js [hunk 1]
```diff
@@ -886,10 +886,8 @@ function returnRandomName(gender){
               lastName:""};
 
   if(gender === "male"){
-    fullName.firstName = utils.returnRandomArrayItem(maleFirstNames);
     fullName.lastName = utils.returnRandomArrayItem(humanLastNames);
   }else{
-    fullName.firstName = utils.returnRandomArrayItem(femaleFirstNames);
```
Comment (KingFruit85) [inline names.js]: @HedgehogHolly We want to keep these in the function as they reference the arrays in names.js, **not core.js**

File: names.js [hunk 2]
```diff
@@ -2,309 +2,6 @@ const utils =require('./utils');
 
 //// TODO: add racial names
 
-const maleFirstNames = [
```
Comment (KingFruit85) [inline names.js]: @HedgehogHolly  Also wanted to keep these. The only things to remove were contained in core.js

File: core.js [hunk 3]
```diff
@@ -382,27 +382,6 @@ function returnRandomCharacterClass() {
 
 
 // TODO: pull from bigger list in new js file
```
Comment (KingFruit85) [inline core.js]: @HedgehogHolly Removed the arrays and functions, perfect. however might as well remove the comment on line 384, that did relate to the removed code.

**Review by KingFruit85 — CHANGES_REQUESTED** [review]
@HedgehogHolly let me know if my comments need to be clarified 🧙‍♂️

File: names.js (hunk 2 above)
Comment (HedgehogHolly) [reply names.js]: Oops!

File: names.js (hunk 1 above)
Comment (HedgehogHolly) [reply names.js]: Added them back in :)

**Comment by HedgehogHolly** [comment]
@KingFruit85, have another look? Ive added the names and function back into name.js

**Comment by KingFruit85** [comment]
> @KingFruit85, have another look? Ive added the names and function back into name.js

@HedgehogHolly  Perfecto, Happy for you to merge :)

**Review by KingFruit85 — APPROVED** [review]

**Pull request merged**